lib/resources/example_json/r4_examples_care_team.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+
+    R4_CARE_TEAM_ENTRY ||= {
+      'resourceType': 'CareTeam',
+      'id': 'SA-1316020-2076906-7016273-0-0-0',
+      'meta': {
+        'versionId': '1',
+        'lastUpdated': '2016-03-10T13:43:45.000-06:00'
+      },
+      'text': {
+        'status': 'additional',
+        'div': '&lt;div xmlns="http://www.w3.org/1999/xhtml"&gt;&lt;p&gt;&lt;b&gt;Care Team&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Patient&lt;/b&gt;: new, testfa2&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: Active&lt;/p&gt;&lt;p&gt;&lt;b&gt;Category&lt;/b&gt;: Encounter-focused care team&lt&lt;/p&gt;&lt;p&gt;Encounter&lt;/b&gt;: 2076906&&lt;/p&gt;lt;p&gt;&lt;b&gt;Participant&lt;/b&gt;:&lt;/p&gt;&lt;ul&gt;&lt;li&gt;Name:, Parimi Vaishnavi; Role:Respiratory Therapy&lt;/li&gt;&lt;/ul&gt;&lt;/div&gt;' # rubocop:disable Layout/LineLength
+      },
+      'status': 'active',
+      "category": [
```

Comment:
use single quotes

Reply:
Addressed https://github.com/cerner/fhir.cerner.com/pull/470/commits/6920a53fa4489aa0584dd2458c3d1517bbf629ff

Reply:
with single quotes, the inner double quotes shouldn't need to be escaped

Reply:
Addressed https://github.com/cerner/fhir.cerner.com/pull/470/commits/31181ae509682c786080e6ef354f2c771f7a5bf2

Reply:
i don't see a double quote fix in that commit. here's what they look like when they are escaped with backslashes:

<img width="525" alt="Screen Shot 2020-10-07 at 1 37 50 PM" src="https://user-images.githubusercontent.com/5505772/95378792-079b1a80-08aa-11eb-95b6-0b81dde11807.png">

were you able to compile the site locally to check?

i pushed the fix with https://github.com/cerner/fhir.cerner.com/pull/470/commits/d035801ecfcbf522dfa51be6884f1011b582c086#diff-f7c10b53dd7ead0281cf4dbd1281a12cR15